config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -57,7 +57,7 @@ presubmits:
           - name: GINKGO_FOCUS
             value: "Workload cluster creation"
           - name: GINKGO_SKIP
-            value: "Creating a GPU-enabled cluster|.*Windows.*dockershim.*|.*AKS.*|Creating a cluster that uses the external cloud provider"
+            value: "Creating a GPU-enabled cluster|.*Windows.*dockershim.*|.*AKS.*|Creating a cluster that uses the in-tree cloud provider"
```

Comment:
@jackfrancis is this PR still needed? #25307 should cover this no?

Reply:
Indeed I didn't realize at the time that we won't need to update test-infra anymore when we finally finish the inversion of "in-tree <---> out-of-tree" capz templates.